quantum/quantum_keycodes.h

```diff
@@ -577,6 +577,13 @@ enum quantum_keycodes {
     ONESHOT_DISABLE,
     ONESHOT_TOGGLE,
 
+    // Tapping term setup
+#ifdef TAP_TERM_KEYS_ENABLE
+    KC_TAPP,
+    KC_TAPU,
+    KC_TAPD,
+#endif
```

Comment:
```suggestion
    TT_PRNT,
    TT_UP,
    TT_DOWN,
```

Reply:
Won't that be confused with the TT of Tap-Toggles?

Reply:
If we can remove the `KC_` prefix, `TAPT_RP`, `TAPT_UP` and `TAPT_DN` would be better candidates imo. It also has the perk to fit in 7 chars.

Reply:
I'd like to keep keycode prefixes to two characters. I guess `TT` could be *slightly* confusing so how about `TK` instead?

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/11036/commits/befc6dea739a2ae59392a6ee98dc8911b55ab9ac